schemas/v3/ArchiveMap.json

```diff
@@ -0,0 +1,189 @@
+                    "description": "The list of the files included in the assessment archive.",
+                    "items": {
+                        "$ref": "#/definitions/FileInfo"
+                    }
+                }
+            }
+        },
+        "FileInfo": {
+            "$id": "#FileInfo",
+            "type": "object",
+            "title": "FileInfo",
+            "description": "Information about a given file within the archive.",
+            "additionalProperties": false,
+            "properties": {
+                "filename": {
+                    "type": "string",
+                    "description": "The filename of the archive object. This should be unique within the manifest.",
+                    "format": "uri-relative"
+                },
+                "description": {
+                    "type": "string",
+                    "description": "Description of the file content."
+                },
+                "contentType": {
+                    "type": "string",
+                    "description": "The content type of the file.",
+                    "default": "application/json"
+                },
+                "jsonSchema": {
+                    "type": "string",
+                    "description": "The uri for the json schema if the content type is 'application/json'.",
+                    "format": "uri"
+                },
+                "isRequired": {
+                    "type": "boolean",
+                    "description": "Whether or not the file is required.",
+                    "default": false
+                },
+                "deprecated": {
+                    "type": "boolean",
+                    "description": "Whether or not the file is deprecated and included for reverse-compatibility to older services and/or applications.",
+                    "default": false
+                },
+                "$comment": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "filename"
+            ]
+        },
+        "AppInfo": {
+            "$id": "#AppInfo",
+            "type": "object",
+            "title": "AppInfo",
+            "description": "Bridge App Information",
+            "additionalProperties": false,
+            "properties": {
+                "appId": {
+                    "type": "string",
+                    "description": "Bridge 'APP ID' for the application that is sending the data."
+                },
+                "iOS": {
+                    "type": "integer",
+                    "description": "The min version sent to Bridge to identify the version of the iOS application.",
+                    "default": 0
+                },
+                "android": {
+                    "type": "integer",
+                    "description": "The min version sent to Bridge to identify the version of the Android application.",
+                    "default": 0
```

Comment:
Do we want to figure out whether we should be including the Bridge app version or the semantic app version and reflect that in this schema?

Reply:
Android apps have an integer based versionCode, so this looks good for Android.

Reply:
I think the question is more about what metadata is in the archive.

Reply:
I'm looking at two records that were created since yesterday and the iOS app version is 78 and the Android app version is 18. These can be found in the `clientInfo` metadata Bridge exports with the data to Synapse. Is this the same "version" which I'm seeing here?

Reply:
yes, those look fine. the iOS version seems low (currently on 88) but that's the idea. doesn't use semantic versioning.

Reply:
Perfect.